docs/UserGuide.md

```diff
@@ -275,14 +345,18 @@ Adds an order for a specified item to an existing member in the loyalty program,
 Format: `addorder n/MEMBER_NAME i/ITEM_NAME [q/QUANTITY_PURCHASED]`
 
 * `ITEM_NAME` will be checked in the catalogue; if it exists, the corresponding item points will be added to the member's current points and membership points.
-* `QUANTITY_PURCHASED` is optional and must be a positive integer; if not specified, it defaults to 1. Currently, the limit is set to 2 billion.
+* `QUANTITY_PURCHASED` is optional and must be a positive integer; if not specified, it defaults to 1.
 
 Examples:
 * `addorder n/John Doe i/Chocolate Cake q/2` Adds `2` orders of `Chocolate Cake` to `John Doe`, updating points accordingly.
 
-🌟 **Benefit**: Streamline the process of tracking purchases and rewarding members by using addorder to add their orders directly into the system. This automatically adjusts their points according to the purchase, fostering a more engaging loyalty experience.<br>
-💡 <span style="color:yellow">**Tip**:</span> Use unique attributes of members when adding points to avoid errors. Partial names are matched to the closest resembling member.<br>
-In the case of similar names like `Betsy Crowead` and `Betsy Croweas`, when adding order to `Betsy Crowe`, the order would be added to the member who was added first in the list.
+<div style="border: 1px solid #D1478C; margin: 10px 0; padding: 10px; background-color: #FDE4EF; color: #910D47; border-radius: 5px;">
+  🧁 <strong>Benefit:</strong> Streamline the process of tracking purchases and rewarding members by using addorder to add their orders directly into the system. This automatically adjusts their points according to the purchase, fostering a more engaging loyalty experience.<br>
+</div>
+
+<div style="border: 1px solid #FFA500; margin: 10px 0; padding: 10px; background-color: #FFE8B2; color: #CC6C00; border-radius: 5px;">
+  ⚠️ <strong>Warning:</strong> You are strongly recommended to only use quantities less than 100 to ensure the total points/membership points a member has remains within the safe range of 1…2,000,000,000.
```

Comment:
Don't think need to mention the 100. Instead we should mention that quantity itself should not exceed safe range of 2,000,000,000